docs/4.tools/near-api-js/using-near-api-js.md

```diff
@@ -0,0 +1,228 @@
+---
+id: using-near-api-js
```

Comment:
Can you keep the original id of this page? the `quick-reference` URL is already updated in many other repositories and external links, and I'd prefer to avoid redirects as much as possible.

Reply:
Ok i changed all back to quick-reference